.github/workflows/release-docker.yml

```
@@ -70,8 +70,8 @@ jobs:
       - name: Get docker build arg
         id: build_arg
         run: |
-          echo "::set-output name=build_arg_name::"$(echo ${{ github.event.inputs.package }} | sed 's/\-/_/g')
-          echo "::set-output name=build_arg_value::"$(echo ${{ github.event.inputs.package }} | sed 's/postgres/core/g')
+          echo "build_arg_name="$(echo ${{ github.event.inputs.package }} | sed 's/\-/_/g') >> $GITHUB_OUTPUT
+          echo "build_arg_value="$(echo ${{ github.event.inputs.package }} | sed 's/postgres/core/g') >> $GITHUB_OUTPUT
```

It's a bit hard to see here, but the format should be
`echo "key=value" >> $GITHUB_OUTPUT`, with the double quotes around the thing being echoed.
```suggestion
          echo "build_arg_name=$(echo ${{ github.event.inputs.package }} | sed 's/\-/_/g')" >> $GITHUB_OUTPUT
          echo "build_arg_value=$(echo ${{ github.event.inputs.package }} | sed 's/postgres/core/g')" >> $GITHUB_OUTPUT
```